packages/executors/http/tests/retry-timeout.test.ts

```diff
@@ -111,7 +111,7 @@ describe('Retry & Timeout', () => {
     expect(result).toMatchObject({
       errors: [
         {
-          message: expect.stringMatching(/timeout/),
+          message: expect.stringMatching(/out/),
```

Comment:
Just `out`? Kinda weird stringmatch. Is it because of case sensitivity or "time out" vs "timeout"?

Reply:
It is `timed out`  in Bun, and `timeout` in Node.

Reply:
Replaced it with better regexp